Zotero/Scenes/Detail/PDF/PDFCoordinator.swift

```diff
@@ -341,19 +345,56 @@ extension PDFCoordinator: PdfReaderCoordinatorDelegate {
         self.navigationController?.present(navigationController, animated: true, completion: nil)
     }
 
-    func showPdfExportSettings(sender: UIBarButtonItem, userInterfaceStyle: UIUserInterfaceStyle, completed: @escaping (PDFExportSettings) -> Void) {
-        DDLogInfo("PDFCoordinator: show PDF export")
-        let view = PDFExportSettingsView(settings: PDFExportSettings(includeAnnotations: true), exportHandler: { [weak self] settings in
-            self?.navigationController?.dismiss(animated: true, completion: {
-                completed(settings)
-            })
-        })
-        let controller = UIHostingController(rootView: view)
-        controller.overrideUserInterfaceStyle = userInterfaceStyle
-        controller.preferredContentSize = CGSize(width: 400, height: 140)
-        controller.modalPresentationStyle = UIDevice.current.userInterfaceIdiom == .pad ? .popover : .formSheet
-        controller.popoverPresentationController?.barButtonItem = sender
-        self.navigationController?.present(controller, animated: true)
+    func showCitation(for itemId: String, libraryId: LibraryIdentifier) {
```

Comment:
You don't have to re-implement this function. You can just call `(parentCoordinator as? DetailsCoordinator)?.showCitation(for: itemId, libraryId: libraryId)`

Reply:
Sure, you can make a common `func showCitation(for itemIds: Set<String>, libraryId: LibraryIdentifier, coordinatorDelegate: DetailCitationCoordinatorDelegate)` and call that from both coordinators.

Reply:
Check https://github.com/zotero/zotero-ios/pull/827/commits/6b4670aa6acab125bde828bfbe392feb681253dd